docs/source/pretty.rst

```diff
@@ -237,4 +237,4 @@ If you want to auto-generate the angular type of repr, then set ``angular=True``
 Example
 -------
 
-See `repr.py <https://github.com/willmcgugan/rich/blob/master/examples/repr.py>`_ for the example code used in this page.
\ No newline at end of file
+See `repr.py <https://github.com/willmcgugan/rich/blob/master/examples/repr.py>`_ for the example code used in this page.
```

Comment:
Not sure what the change is here. It’s from GH’s online editor. Please review.

Reply:
OK, in a real editor 🙂 it had added the final newline. I've put that in a second commit, with some trailing whitespace edits. I leave it to you if you take that or not. (Happy either way). Thanks!